src/components/sidebar-collection/sidebar-collection.jsx

```diff
@@ -31,9 +31,11 @@ class SidebarCollection extends PureComponent {
    * Handle drop collection.
    */
   onDrop = () => {
-    const databaseName = this.props.database;
-    const collectionName = this.getCollectionName();
-    this.props.globalAppRegistryEmit('open-drop-collection', databaseName, collectionName);
+    if (!this.isNotWritable()) {
```

Comment:
Can it be that: 

```
if(this.isNotWritable()) {
return;
}
```

reads better than the not not..?

Reply:
Eh, I'm not sure it really matters